modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/TransactionBuilder.kt

```diff
@@ -26,13 +28,15 @@ class TransactionBuilder(builder: TransactionBuilder.() -> Unit = {}) {
     val instructions: Lazy<ArrayList<Instruction>>
     var creationTimeMillis: BigInteger?
     var timeToLiveMillis: BigInteger?
+    var nonce: Long?
     var metadata: Lazy<HashMap<String, Value>>
 
     init {
         accountId = null
         instructions = lazy { ArrayList() }
         creationTimeMillis = null
         timeToLiveMillis = null
+        nonce = Random.nextLong(0..4_294_967_295) // UInt32 max value
```

Comment:
lets make it constant member based on power of 2, not just literal 

`const val U32_MAX_VALUE = (2L shl 31) - 1`